Resources/Prototypes/_Shitmed/Entities/Structures/Walls/walls.yml

```
@@ -82,5 +82,5 @@
     key: walls
     base: state
   - type: Appearance
-  - type: Reflect
-    reflectProb: 1
+  #- type: Reflect # Mono - remove reflect
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)